index.js

```diff
@@ -1,3 +1,5 @@
+import CharacterCard from "./CharacterCard.js";
```

Comment:
Please check with your team mate, how exactly the `CharacterCard` was exported in the `CharacterCard.js`. You will need to update this import statement. :)